runtime/src/xcm_config.rs

```diff
@@ -76,8 +77,9 @@ pub type XcmOriginToTransactDispatchOrigin = (
 
 parameter_types! {
 	// One XCM operation is 1_000_000_000 weight - almost certainly a conservative estimate.
-	pub UnitWeightCost: u64 = 1_000_000_000;
+	pub UnitWeightCost: Weight = Weight::from_parts(1_000_000_000, 64 * 1024);
```

Comment:
what is this 64k?

Reply:
Proof size

Reply:
is it constant?

Reply:
From what I understand it's some kind of a rough upperbound